readme.md

```diff
@@ -1,4 +1,4 @@
-# Readme.md
+# Roslyn API Reference Docs
```

Comment:
nit: Should this be sentence case?

Reply:
I basically used what we had in other repos but I just changed.